ha/const.py

```diff
@@ -93,6 +95,9 @@
 PCS_CLUSTER_STATUS="pcs cluster status"
 PCS_CLUSTER_UNSTANDBY="pcs cluster unstandby --all"
 PCS_STATUS_NODES="pcs status nodes"
+PCS_NODE_UNSTANDBY="pcs node unstandby <node>"
+PCS_NODE_CLEANUP="pcs resource cleanup --node <node>"
```

Comment:
Is this command available for a specific node. I thought that it's not there?

Reply:
Yes